client/src/components/Modals/SubmitPitch/index.tsx

```diff
@@ -66,60 +77,63 @@ const SubmitPitchModal: FC<SubmitPitchModalProps> = ({
   const [title, setTitle] = useState('');
   const [description, setDescription] = useState('');
   const [link, setLink] = useState('');
-  const [topics, setTopics] = useState(new Set<string>());
+  const [interests, setInterests] = useState<string[]>([]);
   const [conflictofInterest, setConflictOfInterest] = useState<
     boolean | undefined
   >();
```

Comment:
This can just be typed as 

```tsx
const [conflictofInterest, setConflictOfInterest] = useState<boolean>();
```

Leavin the default value blank adds a default 

```tsx 
| undefined
``` 
to the type